core/common/src/main/java/alluxio/exception/status/NotFoundRuntimeException.java

```diff
@@ -22,12 +22,20 @@
 public class NotFoundRuntimeException extends AlluxioRuntimeException {
   private static final Status STATUS = Status.NOT_FOUND;
   private static final ErrorType ERROR_TYPE = ErrorType.User;
-
+  private static final boolean RETRYABLE = true;
```

Comment:
Not found is probably not always retryable? Feels like it should be for the most part not retryable